blog/_posts/2017-09-14-a-song-like-summer.md

```diff
@@ -0,0 +1,124 @@
+---
+title: 🎶 A Song-like Summer 🌞 
+layout: post
+created_at: Thu Sep 14 2017
```

Comment:
Hi, thanks for the PR. Good job on improving many things by your own. Thanks. :) I just have 2 small comments and then blog will be ready to publish. Can you please change a date to Tue September 19, 2017 everywhere?